### DISCUSSION TITLE
RPW Keyboard not working for connected friends

### DISCUSSION THREAD

**Discussion by Mikaaaay** [m4dEngi/RemotePlayWhatever#149] [General]
There's a recent RPW that was posted 14hours ago and seems like I cannot activate their keyboard nor their mouse since whenever I alt+tab or shift+tab in-game, I can't seem to find RPW's options like before. However, they can play using controller but not everyone has one. Previously keyboard and mouse was enough and we can play Tekken 8 from there. Please help.

**Comment by NaturalPotato** [comment]
Did you ever get this fixed?

**Reply by Mikaaaay** [reply]
Not yet. Still waiting for answers/finding alternatives other than Parsec which is really laggy for my friends.
